openzwavemqtt/models/command_class.py

```diff
@@ -50,3 +51,15 @@ def time_stamp(self) -> int:
     def create_collections(self):
         """Create collections that Node supports."""
         return {"value": ItemCollection(OZWValue)}
+
+    def get_value(self, value_index: ValueIndex) -> bool:
```

Comment:
The return value should be an optional value type.

Reply:
yup, sloppy copy/paste work.